packages/grafana-data/src/types/datasource.ts

```diff
@@ -339,7 +339,7 @@ abstract class DataSourceApi<
 
   getVersion?(optionalOptions?: any): Promise<string>;
 
-  interpolateVariablesInQueries?(queries: TQuery[], scopedVars: ScopedVars | {}): TQuery[];
+  interpolateVariablesInQueries?(queries: TQuery[], scopedVars: ScopedVars): TQuery[];
```

Comment:
this `| {}` typing is unnecessary, ScopedVars already supports this (ie can be an empty object)